README.md

```diff
@@ -28,8 +28,8 @@ On SignPath.io:
 
 On Jenkins:
 
-1. Store the **Trusted Build System Token** in a System Credential (Under Manage Jenkins / Manage Credentials)
-2. Store the API Token(s) in a System Credential so that it is available to the build pipelines of the respective projects
+1. Store the **Trusted Build System Token** in a System Credential (Under Manage Jenkins / Manage Credentials) with the id `SignPath.TrustedBuildSystemToken`
+2. Store the API Token(s) in a System Credential so that it is available to the build pipelines of the respective projects (default id `SignPath.ApiToken`)
```

Comment:
That change comes from me, right? We should probably say "credential" instead of "System credential"

Reply:
That was in the commit which you asked me to cherry-pick